building/build-acis/debian/build.sh

```diff
@@ -5,7 +5,7 @@ source ../common/debian.sh
 
 RELEASE="stretch"
 EXTRA_PACKAGES="wget,curl,ca-certificates,git,realpath,file,less,gnupg,python,python3,iptables,iputils-ping,iputils-arping,iproute2,bzip2,gzip,net-tools,netcat-traditional,dnsutils"
-DEBVER=20171105T160402Z
+DEBVER=20180111T215606Z
```

Comment:
Why are we bumping the version of everything?

Reply:
Because we want to include any updates that debian has made in the past couple of months? I haven't been keeping track, but I'm sure there are at least a couple of security updates we'll want to include.

There's no particular reason to do this now other than "I happened to be adding more containers, and I may as well base them off the latest debian."